### PR TITLE
New contact us form block

### PR DESCRIPTION
**Pull request by richpeters**
This also fixes previous contact form admin bug which led to a creash and adds new django debug toolbar version

https://www.meistertask.com/app/task/dQgrp5Mp/create-additional-fields-in-contact-us-form-block-for-52-lives-schools-page-1

### CODE REVIEW

File: apps/companies/migrations/0005_auto_20170619_1600.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
```
Comment (tomkins) [inline apps/companies/migrations/0005_auto_20170619_1600.py]: Rename this (I know it's not your migration).

File: apps/contacts/admin.py [hunk 2]
```diff
@@ -36,6 +36,44 @@ def has_add_permission(self, request):
...
+        (
+            _('Meta data'), {
+                'classes': ('collapse',),
+                'fields': (
+                    'id', 'created_at',
+                )
+            }
+        ),
+    )
+    readonly_fields = list_display
```
Comment (tomkins) [inline apps/contacts/admin.py]: Could you make all the fields readonly? Feels a bit weird having half editable, half read only.

Comment (hawkz) [inline apps/contacts/admin.py]: list_display is all the fields?

File: apps/companies/migrations/0005_auto_20170619_1600.py (hunk 1 above)
Comment (hawkz) [inline apps/companies/migrations/0005_auto_20170619_1600.py]: I've attempted a tidy here

File: apps/contacts/admin.py (hunk 2 above)
Comment (tomkins) [inline apps/contacts/admin.py]: Check fieldsets.

Submit a form, have a look at the admin. Just feels odd having half editable and half not editable.

**Pull request merged**